doc/api/errors.md

```diff
@@ -442,19 +439,7 @@ Some exceptions are *unrecoverable* at the JavaScript layer. Such exceptions
 will *always* cause the Node.js process to crash. Examples include `assert()`
 checks or `abort()` calls in the C++ layer.
 
-## System Errors
-
-Node.js generates system errors when exceptions occur within its runtime
-environment. These usually occur when an application violates an operating
-system constraint. For example, a system error will occur if an application
-attempts to read a file that does not exist.
-
-System errors are usually generated at the syscall level. For a comprehensive
-list, see the [`errno`(3) man page][].
```

Comment:
Should we really remove the whole description? It still seems partially useful to me?

Reply:
See https://github.com/nodejs/node/issues/26861 for context

Reply:
Didn't see Ruben's comment because it was hidden/resolved. I agree with him.